TourGuide_Horde/29_30_HillsbradFoothills.lua

```diff
@@ -12,6 +31,34 @@ A Battle of Hillsbrad |QID|527|
 A Helcular's Revenge |QID|552|
 A Dangerous! |QID|567|
 
+T The WindWatcher |QID|1791| |C|Warrior| |N|(79.29, 6.99)
+
+A Cyclonian |QID|1712| |C|Warrior| |N|This quest involves killing NPCs level 33-34, return to this quest when you feel comfortable.|
+A Essence of the Exile |QID|1714| |C|Warrior| |N|This quest involves killing NPCs level 38-39, return to this quest when you feel comfortable or purchase the required items from someone. Ideally work your way to Hammerfall while completing this.|
+
+F Undercity |C|Warrior|
+F Grom'gol Base Camp |C|Warrior| |N|Take zeppelin to Grom'Gol|
+f Get the flight path |C|Warrior|
+
+K Bloodscalp Scout|QID|1712| |C|Warrior| |QO|Bloodscalp Tusk: 30/30| |N|Bloodscalp trolls (29.8, 21.6) (34.0, 16.8), purchase Liferoot if need be.| |Z|Stranglethorn Vale|
```

Comment:
What a line! It's missing the |Q| tag to associate with the |QO| tag though :) And watch the space between the title and the tag.